cmd/genji/shell/shell.go

```diff
@@ -213,16 +213,29 @@ func (sh *Shell) executeInput(in string) error {
 	return nil
 }
 
-func (sh *Shell) runCommand(cmd string) error {
-	switch cmd {
+func (sh *Shell) runCommand(in string) error {
+	cmd := strings.Fields(in)
+	switch cmd[0] {
 	case ".tables":
 		db, err := sh.getDB()
 		if err != nil {
 			return err
 		}
-		return runTablesCmd(db)
+
+		return runTablesCmd(db, cmd)
 	case ".exit":
+		if len(cmd) > 1 {
+			return fmt.Errorf("too many arguments in call to %s", cmd[0])
+		}
+
```

Comment:
I think that how it was done before was enough. Could you tell me what motivates you to change this? There is probably a use case that I don't see.

Reply:
See two comments above.